modules/flowable-engine-common/src/main/java/org/flowable/common/engine/impl/el/function/VariableGetOrDefault2ExpressionFunction.java

```diff
@@ -0,0 +1,25 @@
+package org.flowable.common.engine.impl.el.function;
```

Comment:
Please include the license header comment block at the start of this file; thanks.

Reply:
Got it. Apache license header added